src/modules/fundingManager/bondingCurve/FM_BC_BondingSurface_Redeeming_Restricted_Repayer_Seizable_v1.sol

```diff
@@ -167,9 +167,6 @@ contract FM_BC_BondingSurface_Redeeming_Restricted_Repayer_Seizable_v1 is
         // Set issuance token. This also caches the decimals
         _setIssuanceToken(address(_issuanceToken));
 
-        // Set token Vault
-        _setTokenVault(_tokenVault);
-
```

Comment:
Why was this removed?

Reply:
Because we have a dependency injection problem here.
The Token vault is a module. The BondingSurfaceContract is a module.
Both of them are created and initialized essentially at the same time.
That means the BondingSurface cant get the tokenVault address during initialization.
Thats what the setter is for

Reply:
Ok, so why still have in the configData, and why save it in an unused memory variable? Those should be removed as well no?

Reply:
Yes you are correct
Will address that